sh/configure_env.sh

```diff
@@ -161,16 +161,38 @@ LDARGS=$(echo "$LDARGS" | sed -e "s/ CoreFoundation//g")
 # IFS = Internal Field Separator - allows looping over lines with spaces
 IFS=$'\n'
 
-
+ANACONDA=0
+LIBLDARGS=""
 # Add library path if using anaconda python
 PYVER=$($PY --version 2>&1 )
 if [[ "$PYVER" == *"Anaconda"* ]]; then
+  ANACONDA=1
   ANACONDA_LIB_PATH=$(linkread $(dirname $(which $PY))/../lib)
-  LDARGS="-L${ANACONDA_LIB_PATH} -Wl,-rpath,${ANACONDA_LIB_PATH} $LDARGS -lmkl_rt -lpthread"
-  LDARGS="-dynamiclib -undefined dynamic_lookup -Wl,-headerpad_max_install_names"
-  CXXFLAGS="-DSCIPY_MKL_H -DHAVE_CBLAS -DTICK_CBLAS_AVAILABLE $CXXFLAGS"
+  if [[ "$unameOut" == "Darwin"* ]]; then
+    # Unknown reason why these are not required for Anaconda -
+    #  Not just that but neither is the -lpython
+    LDARGS="" #-L${ANACONDA_LIB_PATH} -Wl,-rpath,${ANACONDA_LIB_PATH} $LDARGS -lmkl_rt -lpthread"
+    LIBLDARGS="-dynamiclib -undefined dynamic_lookup -Wl,-headerpad_max_install_names"
+    CXXFLAGS="-DSCIPY_MKL_H -DHAVE_CBLAS -DTICK_CBLAS_AVAILABLE $CXXFLAGS"
+  fi
 fi
 
+MKN_P_ARRAY=(lib_name=$LIB_POSTFIX)
+case "${unameOut}" in
+    Linux*)     LDARGS="${LDARGS}";;
+    Darwin*)    LDARGS="${LDARGS}";;
+    CYGWIN*)    LDARGS="${LDARGS}";;
+    MINGW*)     LDARGS="${LDARGS}";;
+    *)          LDARGS="${LDARGS}";;
+esac
+MKN_P=""
```

Comment:
Why don't we call it `MKN_PROPERTIES` ? It looks easier to understand this way

Reply:
the "-P" argument for mkn refers to "properties

Reply:
https://github.com/X-DataInitiative/tick/pull/113/files#diff-8c1ca1a035c8f4df5bca4bc9814477f9R200